docs/jwst/outlier_detection/outlier_detection_imaging.rst

```diff
@@ -1,180 +1,127 @@
 
        .. math:: | image\_input - image\_median | > SNR * input\_err
 
-   * When resampling is enabled, the comparison uses the original input images, the blotted
-     median image, and the derivative of the blotted image to
-     create a cosmic ray mask for each input image.
-   * The derivative of the blotted image gets created using the blotted
-     median image to compute the absolute value of the difference between each pixel and
-     its four surrounding neighbors with the largest value being the recorded derivative.
-   * These derivative images are used to flag cosmic rays
-     and other blemishes, such as satellite trails. Where the difference is larger
-     than can be explained by noise statistics, the flattening effect of taking the
-     median, or an error in the shift (the latter two effects are estimated using
-     the image derivative), the suspect pixel is masked.
-   * The ``backg`` parameter specifies a user-provided value to be used as the
-     background estimate.  This gets added to the background-subtracted
-     blotted image to attempt to match the original background levels of the
-     original input mosaic so that cosmic-rays (bad pixels) from the input
-     mosaic can be identified more easily as outliers compared to the blotted
-     mosaic.
-   * Cosmic rays are flagged using the following rule:
+   * Add a user-specified background value to the median image to match the original background levels
+     of the input mosaic. This is controlled by the ``backg`` parameter.
+   * Compute the spatial derivative of each pixel in the blotted median images by computing the absolute value
+     of the difference between each pixel and its four surrounding neighbors, recording the largest
+     absolute difference as the derivative. The derivative is multiplied by the ``scale`` parameter,
+     allowing user flexibility in determining how aggressively to flag outliers at this stage.
+   * Flag cosmic rays and other blemishes (such as satellite trails) based on the derivative image.
+     Where the difference is larger than can be explained by a combination of noise statistics,
+     the flattening effect of taking the median, and an error in the shift
+     (the latter two effects are estimated using the image derivative), the suspect pixel is considered
+     an outlier. The following rule is used:
 
      .. math:: | image\_input - image\_blotted | > scale*image\_deriv + SNR*noise
 
-   * The ``scale`` is defined as the multiplicative factor applied to the
-     derivative which is used to determine if the difference between the data
-     image and the blotted image is large enough to require masking.
-   * The ``noise`` is calculated using a combination of the detector read
+     The noise in this equation is calculated using a combination of the detector read
      noise and the poisson noise of the blotted median image plus the sky background.
```

Comment:
I don't think this is correct - the noise in the equation is the input error for the image.

Reply:
How is that computed?

I agree this should be changed, because the outlier step doesn't do any of those calculations, but I think the intent was to re-explain how we got that error estimate in the first place

Reply:
It is normal error propagation for everything up to this point in the pipeline - the error estimate is a combination of the flat, read noise, and poisson variance components.  The important part here is that it is not from the "blotted median image plus the sky background". It is the error estimate corresponding to the input SCI array, unresampled and unmasked.